### ISSUE TITLE
Idyll does not build (Node 16 issue?)

### ISSUE THREAD

**Issue by vilim** [idyll-lang/idyll#752]
**Describe the bug**
Compiling a new project fails with error 
```
.../my-idyll-post/node_modules/scrollmonitor/dist/index.js:1
export { ScrollMonitorContainer } from './src/container.js';
^^^^^^

SyntaxError: Unexpected token 'export'
```
I got Idyll to run once, but I think the node.js version was different then (12.x)

**To Reproduce**
1. Follow install and new project instructions from the idyll docs
2. run the `idyll` command in the newly-created project
the build error occurs

**Desktop (please complete the following information):**
 - OS: Ubuntu Linux 21.04
 - Node.js: v16.13.0
 - npm: 7.5.3

**Additional context**
Full error message:
```
.../my-idyll-post/node_modules/scrollmonitor/dist/index.js:1
export { ScrollMonitorContainer } from './src/container.js';
^^^^^^

SyntaxError: Unexpected token 'export'
    at Object.compileFunction (node:vm:352:18)
    at wrapSafe (node:internal/modules/cjs/loader:1031:15)
    at Module._compile (node:internal/modules/cjs/loader:1065:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Object.require.extensions.<computed> [as .js] (.../interactive/my-idyll-post/node_modules/babel-register/lib/node.js:152:7)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.Module._load (.../my-idyll-post/node_modules/idyll/src/node-config.js:23:29)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (.../my-idyll-post/node_modules/idyll-document/dist/cjs/runtime.js:19:22)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Object.require.extensions.<computed> [as .js] (..//my-idyll-post/node_modules/babel-register/lib/node.js:152:7)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    ```

**Comment by mathisonian** [comment]
@vilim thanks for reporting this - I am able to reproduce this locally and can confirm its a bug. It affects at least node v14 as well, so it may not be a node version issue. I'm going to dig into this and will update with possible fixes / workarounds as I learn more.

**Comment by mathisonian** [comment]
This seems to be an issue with `scrollmonitor@1.2.7` specifically. Here's a workaround until I can get a patch released:

```
$ idyll create
...
$ cd my-new-post
$ npm i --save scrollmonitor@1.2.6
$ idyll
```

Basically just pin scrollmonitor to the older release and everything should build fine

**Comment by vilim** [comment]
Thank you!

**Comment by mathisonian** [comment]
No problem! This should be fixed in the latest release of Idyll (`idyll@5.7.3` just released on npm) with no workaround needed. Feel free to re-open if you continue to have issues.
